N/neper/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+using BinaryBuilder, Pkg
+
+name = "neper"
+version = v"4.5.0"
+
+sources = [
+    ArchiveSource("https://github.com/neperfepx/neper/archive/refs/tags/v$(version).tar.gz", "db80dd89e02207e9b056b05fb9fbe493199ce7c3736b2039104c595b4dcd02a9")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/neper-*
+mkdir build && cd build
+CXXFLAGS="-lz"
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
+      -DCMAKE_INSTALL_PREFIX_COMPLETION_FULL=$prefix/completion \
+      ../src
+make -j${nproc}
+make install
+"""
+
+function exclude(p)
+    if get(p.tags, "libc", nothing) == "musl" || get(p.tags, "os", nothing) == "freebsd"
+        # src/contrib/ut/src/ut_print/ut_print.c:1685:38: error: parameter 1 (‘beg_time’) has incomplete type
+        #  1685 | ut_print_elapsedtime (struct timeval beg_time, struct timeval end_time)
+        #       |                       ~~~~~~~~~~~~~~~^~~~~~~~
+        return true
```

Comment:
This can probably be fixed with a patch, but I am not experienced enough with C to do that I'm afraid.